project/plugins.sbt

```diff
@@ -1,2 +1,4 @@
 addSbtPlugin("org.http4s" % "sbt-http4s-org" % "0.14.3")
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.10.0")
+// Fix links to API docs of scala-xml
+addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "3.0.2")
```

Comment:
Blah, I don't like to see this dependency added. The problem should be fixed upstream.

Reply:
- https://github.com/scala/scala-xml/pull/620

Reply:
I was about to read up on the details of API doc linking…thank you for handling this so quickly!

Reply:
I was about to read up on the details of API doc linking…thank you for handling this so quickly!